tunnel-registrator.service

```diff
@@ -13,7 +13,7 @@ ExecStart=/bin/bash -c '\
   KONSTRUCTOR_API_KEY=$(/usr/bin/etcdctl get /ft/_credentials/konstructor/api-key); \
   PRIVATE_IP=$(curl -s 169.254.169.254/latest/meta-data/local-ipv4); \
   docker run --rm --name %p_$(uuidgen) \
-  -e="DOMAINS=$ENV-neo4j-tunnel-up" \
+  -e="DOMAINS=$ENV-tunnel-up" \
```

Comment:
We're changing the environment tag to not be the same as the delivery cluster.  The new tag will be *-data (e.g. pre-prod-uk-data), so it wouldn't overwrite.